api/models/call.go

```diff
@@ -146,9 +136,16 @@ type Call struct {
 	Error string `json:"error,omitempty" db:"error"`
 }
 
+// Call is a representation of a specific invocation of a route.
+type Call struct {
+	CallBase
+	// App this call belongs to.
+	AppID string `json:"app_id" db:"app_id"`
```

Comment:
should we hide this in json but leave it in the db (and on route, logs, etc.)? part of the evil plan was to hide app id to user. to address these resources they'll be supplying app name. we can remove it later, but it's worth discussion at least

Reply:
fixed